modules/solace/options.go

```diff
@@ -0,0 +1,105 @@
+package solace
+
+import (
+	"fmt"
+
+	"github.com/testcontainers/testcontainers-go"
+)
+
+type options struct {
+	vpn          string
+	username     string
+	password     string
+	exposedPorts []string
+	queues       map[string][]string // queueName -> topics
+	envVars      map[string]string
```

Comment:
suggestion: I think we do not need the envVars here, as we have a `testcontainers.WithEnv` option in the core.